src/main/java/com/javadiscord/jdi/internal/api/DiscordRequestBuilder.java

```diff
@@ -0,0 +1,84 @@
+package com.javadiscord.jdi.internal.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DiscordRequestBuilder {
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private String method;
+    private String url;
```

Comment:
put the initial value `https://discord.com/api`

Reply:
Let's see how it pans out and leave this as future tech debt

Reply:
as of now its not really the URL is it?

Reply:
yes then dont name it url

Reply:
suggestion?

Reply:
split it into `path` and `parameters`, for long path + parameters it'll get messy anyways

Reply:
:S idk how to do this cleanly

Reply:
i'll do it later, dw wont change much of ur code

Reply:
Thanks - if everything is all good, can you approve? :)